### PR TITLE
Added booking detail view in counterparties

### PR DESCRIPTION
**Pull request by cyberkarma**
Added booking detail view in counterparties

### CODE REVIEW

File: src/core/routes.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ import createPaymentLinkPage from 'pages/paymentlinks/create';
 import inviteAdminPage from 'pages/admins/invite';
 import duplicatesListPage from 'pages/duplicates/list';
 import duplicatePage from 'pages/duplicates/duplicate';
+import Booking from '../parts/bookings/booking';
```
Comment (alexander-plekhanov) [inline src/core/routes.js:15]: import bookingPage from 'pages/counterparty/booking'

File: src/core/routes.js [hunk 2]
```diff
@@ -24,6 +25,7 @@ const Routes = () => (
         <Route exact path="/admins" component={inviteAdminPage} />
         <Route exact path="/duplicates/:id" component={duplicatePage} />
         <Route exact path="/duplicates" component={duplicatesListPage} />
+        <Route exact path="/counterparties/booking/:id" component={Booking} />
```
Comment (alexander-plekhanov) [inline src/core/routes.js:28]: "/booking/:refCode"

File: src/parts/bookings/booking.js [hunk 3]
```diff
@@ -37,21 +57,25 @@ class Booking extends React.Component {
...
-                        <button className="button" onClick={this.paymentConfirm}>Confirm Payment</button>
-                    </div>
-                    <div className="buttons">
-                        <button className="button" onClick={this.props.onClose}>Back</button>
+                    <div style={{ display: 'flex', justifyContent: 'space-between', margin: '50px 0 30px' }}>
+                        <div className="buttons">
+                            <button className="button" onClick={this.bookingCancel}>Cancel</button>
+                            <button className="button" onClick={this.bookingDiscard}>Discard</button>
+                            <button className="button" onClick={this.bookingPaymentCompleteManually}>Manually Complete
+                                Payment
```
Comment (alexander-plekhanov) [inline src/parts/bookings/booking.js:69]: line break issue:
< tag >
   ____ text
< /tag >

File: src/parts/bookings/booking.js [hunk 4]
```diff
@@ -37,21 +57,25 @@ class Booking extends React.Component {
...
+                            </button>
+                            <button className="button" onClick={this.paymentConfirm}>Confirm Payment</button>
+
```
Comment (alexander-plekhanov) [inline src/parts/bookings/booking.js:72]: don't need extra space lines before closing tag

**Review by alexander-plekhanov — APPROVED** [review]

**Comment by cyberkarma** [comment]
confirm

**Pull request merged**